client/components/happychat/button.jsx

```diff
@@ -15,28 +17,47 @@ import { openChat } from 'state/ui/happychat/actions';
 import Button from 'components/button';
 
 class HappychatButton extends Component {
-	onOpenChat = () => {
-		const { onOpenChat } = this.props;
+	static propTypes = {
```

Comment:
I sorted the props but I don't think we need to add `className` here since its kinda an intrinsic prop.